src/khun/KhunExtension.cs

```diff
@@ -16,6 +17,11 @@ public static class KhunExtension {
         "อ."
       };
 
+    private static string[] vowels = new string[] {
```

Comment:
ถ้าอย่างนั้น เราน่าจะมี test cases ได้ประมาณนี้

// names that in **the list**
"คุณวุฒิ".AsKhun() => "คุณคุณวุฒิ"
"คุณคุณวุฒิ".AsKhun() => "คุณคุณวุฒิ"

// names that not in **the list** (can check with vowels)
"คุณากร".AsKhun() => "คุณคุณากร"
"คุณคุณากร".AsKhun() => "คุณคุณากร"

// names that not in **the list** (cannot check with vowels)
"คุณศรี".AsKhun() => "คุณศรี"
"คุณคุณศรี".AsKhun() => "คุณคุณศรี"

// the list
string[] specialNames = new string [] { "คุณวุฒิ", "คุณสมบัติ" };